docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,9 @@ Version history
 
 1.15.0 (Pending)
 ================
+* tracing: tracing provider configuration as part of :ref:`bootstrap config <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.tracing>`
```

Comment:
nit: I wouldn't discuss deprecation here but rather just describe the new way of configuring it. You describe deprecation elsewhere.

Reply:
reworded